tasks/perun_openldap_config.yml

```diff
@@ -132,51 +132,119 @@
+            create: yes
+            block: |
+              [Service]
+              LimitNOFILE="{{ perun_ldap_no_file_limit_value }}"
+      register: ldap_nofile_limit
+      
+    - name: "reload systemd to apply override changes"
+      when:  soft_limit.changed or hard_limit.changed or fs_file_max
+      ansible.builtin.systemd:
+            daemon_reload: true
+    
+    - name: "restart slapd service to apply new limits"
+      when: ldap_nofile_limit.changed
+      ansible.builtin.service:
+            name: slapd
+            state: restarted
+
+    - name: "get soft open file limit for openldap"
+      ansible.builtin.command: "grep '^openldap.*soft.*nofile' /etc/security/limits.conf"
+      register: ulimit_soft_result
+      when: soft_limit.changed
+
+    - name: "get hard open file limit for openldap"
+      ansible.builtin.command: "grep '^openldap.*hard.*nofile' /etc/security/limits.conf"
+      register: ulimit_hard_result
+      when: hard_limit.changed
+
+    - name: "get system max open files limit"
+      ansible.builtin.command: "sysctl fs.file-max"
+      register: sysctl_result
+      when: fs_file_max.changed
+
+    - name: "get slapd LimitNOFILE setting"
+      ansible.builtin.command: "systemctl show -p LimitNOFILE slapd"
+      register: limitnofile_result
+      when: ldap_nofile_limit.changed
+
+    - name: "display configured soft open file limit for openldap"
+      when: soft_limit.changed
+      debug:
+        msg: "Configured soft open file limit for openldap: {{ ulimit_soft_result.stdout }}"
+
+    - name: "display configured hard open file limit for openldap"
+      when: hard_limit.changed
+      debug:
+        msg: "Configured hard open file limit for openldap: {{ ulimit_hard_result.stdout }}"
+
+    - name: "display system max open files limit (fs.file-max)"
+      when: fs_file_max.changed
+      debug:
+        msg: "System max open files limit (fs.file-max): {{ sysctl_result.stdout }}"
+
+    - name: "display slapd LimitNOFILE setting"
+      when: ldap_nofile_limit.changed
+      debug:
+        msg: "slapd LimitNOFILE setting: {{ limitnofile_result.stdout }}"
+
+
+- name: "set up defaults limits for max open files for instance"
+  when: not perun_ldap_max_open_files_limits
+  block:
+  - name: "remove soft and hard open file limits for openldap user"
+    ansible.builtin.lineinfile:
+      path: /etc/security/limits.conf
+      state: absent
+      regexp: '^openldap.*nofile'
+
+  - name: "reset system-wide open files limit to default"
+    ansible.posix.sysctl:
+      name: fs.file-max
+      state: absent
+
+  - name: "remove systemd override config directory for slapd"
+    ansible.builtin.file:
+      path: /etc/systemd/system/slapd.service.d/override.conf
+      state: absent
+
+  - name: "remove systemd override directory if empty"
+    ansible.builtin.file:
+      path: /etc/systemd/system/slapd.service.d
+      state: absent
```

Comment:
I think we should restart relevant services when we remove the limits just like we do when we set them. E.g.:
```yaml

    - name: "remove systemd override directory if empty"
      ansible.builtin.file:
        path: /etc/systemd/system/slapd.service.d
        state: absent
      register: ldap_nofile_limit

    - name: "restart slapd service to apply new limits"
      when: ldap_nofile_limit.changed
      ansible.builtin.service:
            name: slapd
            state: restarted
```